src/ast/ATTRIBUTES.ts

```diff
@@ -0,0 +1,22 @@
+import NODE from "./NODE";
+import ATTRIBUTE from "./ATTRIBUTE";
+
+export default class ATTRIBUTES extends NODE {
+  attributes: ATTRIBUTE[] = [];
+  parse() {
+    const areAttributes = this.tokenizer.checkToken(
+      "add Color|add Alignment|add Direction"
+    );
+    if (areAttributes) {
+      while (
+        this.tokenizer.moreTokens() &&
+        !this.tokenizer.checkToken("add Tags") &&
```

Comment:
Just a consideration of treating Tags as just another attribute since it now shares the same syntax and would make the code a little cleaner. But it's up to you if you wanna do this refactoring

Reply:
I just figured it would make more sense to split them because tags is Tag+ while attributes is Attribute? (actually needs to check between predefined values - I will add that). I can change it to a single file if everyone would really prefer though 😅